### ISSUE TITLE
Inconsistency in the generator architecture 

### ISSUE THREAD

**Issue by krantiparida** [haoheliu/voicefixer#29]
Thanks for releasing the code publicly. I have a small confusion in the implementation of the generator mentioned [here](https://github.com/haoheliu/voicefixer/blob/1bc47bebac20c365fda0a6e93b5f04ebb1ee1a1c/voicefixer/restorer/model.py#L103). As per Fig.3(a) in the [paper](https://arxiv.org/pdf/2109.13731.pdf), a mask is predicted from the input noisy audio which is then multiplied with the input to get the clean audio, but in the implementation, it seems the after the masking operation it is further passed through a unet. The loss is also calculated for both the outputs. Can you please clarify the inconsistency? Thanks in advance.

**Comment by haoheliu** [comment]
Apologies for the confusion. This repo hosts an early implementation of the voicefixer model. In our later experiments we found the use of denoiser does not affect the result too much. So we simplify the system into the Unet only version in our paper. Hope that make sense to you.

**Comment by krantiparida** [comment]
Thanks for the clarification. 

I have one more confusion. In the implementation of `VoiceFixer` class, there is a variable defined as `self.mel_weight_44k_128` [here](https://github.com/haoheliu/voicefixer/blob/1bc47bebac20c365fda0a6e93b5f04ebb1ee1a1c/voicefixer/restorer/model.py#L219). I guess it is some kind of weights for each mel-frequency bin in the mel-spectrogram to be used for normalization. Can you please clarify what is this used for and how the values are obtained? Thanks.

**Comment by haoheliu** [comment]
You can also refer to [this function](https://pytorch.org/audio/stable/transforms.html#torchaudio.transforms.MelScale). The argument of this function "norm" is related with the frequency response width mentioned above. If you set "norm" to "slaney". The function will perform "area normalization".

BTW: Not performing "Area normalization" when calculating the mel spectrogram is found crucial in voicefixer training. Probably because without area normalization high frequency can receive more weights on calculating loss function.

**Comment by krantiparida** [comment]
Thanks for the clarification.
